clicmd

```diff
@@ -25,8 +81,8 @@ function subcommand {
     print_help "${parent}" | head -n 1
     local cmd
     for cmd in ${children[*]}; do
-      printf "%-16s " "${cmd#${parent}_}"
-      print_help "${cmd}" | head -n 1
+      local help=$(print_help "${cmd}" | head -n 1)
+      [[ -n $help ]] && printf "%-16s %s\n" "${cmd#${parent}_}" "$help"
```

Comment:
`[[ -n "${help}" ]]`
Quotes are mandatory for -n and -z.

Reply:
Ok